src/main/java/domain/RacingGame.java

```diff
@@ -0,0 +1,91 @@
+package domain;
+
+public class RacingGame {
+    private int tryCnt;
+    private Car[] players;
```

Comment:
```suggestion
    private List<Car> players;
```
배열보다는 List를 사용하는 것을 강력 권장드립니다!

배열보다 편하고 강력한 기능들이 많기 때문에, 굳이 배열을 사용할 이유가 없다고 생각해요 ㅎㅎ

Reply:
배열을 사용할 때 당장 크기 지정을 하는 것도 껄끄러웠는데 리스트를 사용하면 더 편리하겠네요..

감사합니다!!